ament_pep257/ament_pep257/main.py

```diff
@@ -45,11 +46,20 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--ignore',
         nargs='*',
-        default=[
-            'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D106', 'D107',
-            'D203', 'D212', 'D404',
```

Comment:
Ultimately, an external user could relax constraints (i.e. ignore both D212 and D213) without us changing the defaults.

Reply:
Restored the default error checking behavior in e09982a.

Note that some merging code is necessary to duplicate the old behavior 

https://github.com/ament/ament_lint/blob/e09982a0e36865efe92dc4f6b53efe74dce8153f/ament_pep257/ament_pep257/main.py#L76-L82

Reply:
Hmm, instead of making assumptions on what the default ignored error codes are and forcing others to set a baseline, I'd much rather replicate `pydocstyle` interface i.e. to have `--ignore`, `--select`, `--add-ignore`, and `--add-select`, and keep the default `--ignore` list as it used to be. Thoughts?